src/client/src/containers/Footer/index.tsx

```diff
@@ -202,30 +202,28 @@ class Footer extends React.Component<Props> {
               >
                 <FormattedMessage id="footer.back" defaultMessage="Back" />
               </Link>
-              <Link
-                tabIndex={
-                  !isValidNameAndProjectPath || this.isReviewAndGenerate()
-                    ? -1
-                    : 0
-                }
-                className={classnames(styles.button, {
-                  [buttonStyles.buttonDark]:
-                    this.isReviewAndGenerate() || !isValidNameAndProjectPath,
-                  [buttonStyles.buttonHighlightedBorder]: !this.isReviewAndGenerate(),
-                  [styles.disabledOverlay]:
-                    !isValidNameAndProjectPath || this.isReviewAndGenerate()
-                })}
-                onClick={event => {
-                  this.handleLinkClick(event, pathname);
-                }}
-                to={
-                  pathname === ROUTES.REVIEW_AND_GENERATE
-                    ? ROUTES.REVIEW_AND_GENERATE
-                    : pathsNext[pathname]
-                }
-              >
-                <FormattedMessage id="footer.next" defaultMessage="Next" />
-              </Link>
+              { pathname !== ROUTES.REVIEW_AND_GENERATE &&
+                <Link
+                  tabIndex={
+                    !isValidNameAndProjectPath ? -1 : 0
```

Comment:
NIT: feels to me like this negation is not necessary here and it would be more readable:  `isValidNameAndProjectPath ? 0 : -1`